platform-common-core/src/main/java/com/softicar/platform/common/core/i18n/I18nFormatArgumentsProcessor.java

```diff
@@ -0,0 +1,38 @@
+package com.softicar.platform.common.core.i18n;
+
+import com.softicar.platform.common.core.locale.CurrentLocale;
+import com.softicar.platform.common.core.number.formatter.BigDecimalFormatter;
+import java.math.BigDecimal;
+
+/**
+ * Transforms I18n formatting arguments according to {@link CurrentLocale}.
+ *
+ * @author Oliver Richers
+ */
+class I18nFormatArgumentsProcessor {
```

Comment:
```suggestion
class I18nFormatArgumentsLocalizer {
```
This class is all about localization. We could (and probably should) express that in its name.

Reply:
ok, renamed